tests/workers/test_credit_processor_mixin.py

```
@@ -69,8 +70,13 @@ async def test_send_with_optional_cancel_success(self, mock_wait_for, mixin):
         async def simple_coroutine():
             return mock_record
 
-        # Mock wait_for to return the successful result
-        mock_wait_for.return_value = mock_record
+        # Mock wait_for to consume the coroutine and return the successful result
+        async def mock_wait_for_impl(coro, timeout):
```

Ruff (0.13.1)

Unused function argument: `timeout`

(ARG001)